tests/test_webapptest.py

```diff
@@ -0,0 +1,22 @@
+"""
+Tests for the WebAppTest class.
+"""
+import os
+from bok_choy.web_app_test import WebAppTest
+from .pages import ImagePage
+
+
+class ScreenshotAssertTest(WebAppTest):
+    """
+    Test the integration with needle and its screenshot assertion capability.
+    """
+    def test_needle_screenshot(self):
```

Comment:
Is there any way to test that assertScreenshot fails when it should fail? I'd like that negative test if possible.